src/gui/editors/PianoRoll.cpp

```diff
@@ -404,6 +405,23 @@ PianoRoll::PianoRoll() :
 	connect( &m_noteLenModel, SIGNAL( dataChanged() ),
 					this, SLOT( noteLengthChanged() ) );
 
+	// Set up key selection
+	m_keyModel.addItem(tr("No key"));
+	m_keyModel.addItem(tr("C"));
+	m_keyModel.addItem(tr("C♮/D♭"));
```

Comment:
Shouldn't this be "C♯/D♭"? Same for the others.

Reply:
doh... I couldn't tell from the font. Fixing.